server/odc-service/src/main/java/com/oceanbase/odc/service/quartz/OdcJobListener.java

```diff
@@ -129,6 +130,12 @@ public void jobExecutionVetoed(JobExecutionContext context) {
     public void jobWasExecuted(JobExecutionContext context, JobExecutionException jobException) {
         JobKey key = context.getJobDetail().getKey();
         List<? extends Trigger> jobTriggers;
+        Optional<ScheduleEntity> scheduleEntityOptional =
+                scheduleRepository.findById(ScheduleTaskUtils.getScheduleId(context));
+        if (!scheduleEntityOptional.isPresent()
```

Comment:
a little hard to understand..

may split into 2 steps

if(!scheduleEntityOptional.isPresent() {
    // if unespected, log.warn here 
    return;
}

isRollbackTask=!key.getGroup().equals(scheduleEntityOptional.get().getJobType().name())
if (isRollbackTask) {
 return;
}

Reply:
simplified the method and the key of the rollback task  not used to find the trigger now.